jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -293,7 +296,7 @@ def rotation_angle_deg(self, rotation_angle=None):
         return 0 * u.deg
 
     def add_orientation(self, rotation_angle=None, east_left=None, label=None,
-                        set_on_create=True, wrt_data=None):
+                        set_on_create=True, wrt_data=None, from_ui=False):

```

Comment:
Since this is a publicly exposed method, is there any user-friendly name we can use here?  Maybe just `require_data`?  Or can we instead wrap with a try/except at the `vue_*` level that converts errors into snackbars and leaves the public methods alone?

Reply:
Hmm. I could also make `add_orientation` just pass the public-facing args to a private `_add_orientation` that has the new keyword for use by the `vue_` method. I'll think about the best way to do this.

Reply:
I moved the new keyword arg to a private method.